lib/Bort.pm

```diff
@@ -345,14 +347,19 @@ sub load_team {
 
 
 sub my_user      { $my_user }
-sub channel_name { shift; my $channel = $_[-1]; $channel_names{$channel} // $channel }
-sub user_name    { shift; my $user = $_[-1]; $user_names{$user} // $user }
+sub channel_name { shift; my $channel = $_[-1]; $channel_names{$channel} // 0 }
+sub user_name    { shift; my $user = $_[-1]; $user_names{$user} // 0 }
 
 sub my_user_name    { $my_user_name }
-sub channel_by_name { $channel_by_name{$_[-1]} // $_[-1] };
-sub user_by_name    { $user_by_name{$_[-1]} // $_[-1] };
-
-sub user_data { shift; my $user = $_[-1]; $user_data{$user} }
+sub channel_by_name { $channel_by_name{$_[-1]} // 0 };
+sub user_by_name    { $user_by_name{$_[-1]} // 0 };
+
+sub user_data {
+  shift;
+  my $user = $_[-1];
+  # Naive user_id validation
+  $user =~ /^U[0-9A-Z]+/ ? $user_data{$user} : 0
```

Comment:
This validation seems unnecessary. Either its `%user_data`, or its not?

Reply:
Discussion: shouldn't be needed. If Slack are going to give us junk (and they're not), we should filter elsewhere.